tests/ServerManager/GameListCreationTest.php

```diff
@@ -96,6 +97,18 @@ public static function setUpBeforeClass(): void
         $input = new ArrayInput([
             'command' => 'doctrine:database:drop',
             '--connection' => 'msp_session_1', // don't worry, only removes msp_session_1_test database
+            '--if-exists' => true,
+            '--force' => true,
+            '--no-interaction' => true,
+        ]);
+        $input->setInteractive(false);
+        $app->doRun($input, new NullOutput());
+
+        $app = new Application(static::bootKernel());
+        $input = new ArrayInput([
```

Comment:
how are you planning to handle https://jira.cradle.buas.nl/browse/MSP-4966


could we use data fixtures ? 
https://symfonycasts.com/screencast/phpunit-legacy/control-database

Reply:
will be in different branch and PR